### PR TITLE
imageFilter (support for the CSS filter attribute)

### PR DESCRIPTION
**Pull request by Fil**
Helps with dark mode (#408)

### CODE REVIEW

File: src/style.js [hunk 1]
```diff
@@ -14,6 +15,7 @@ export function Style(mark, {
   strokeDasharray,
   mixBlendMode
 } = {}) {
+  mark.colorFilter = string(colorFilter);
```
Comment (mbostock) [inline src/style.js:18]:
```suggestion
  mark.colorFilter = impliedString(colorFilter, "none");
```

**Review by mbostock — COMMENTED** [review]
I’m going to punt on this. I’m unsure about the name “colorFilter”. It’s unfortunate that we can’t use *filter* because it conflicts with the basic transform. We could, possibly, interpret the *filter* as CSS filter if it’s a valid CSS filter function string, similar to how we validate colors. But that’s a little fancy…

https://drafts.fxtf.org/filter-effects/#filter-functions

**Comment by mbostock** [comment]
I think we should do this, but we should call it **imageFilter** instead of **colorFilter**.

**Comment by Fil** [comment]
rebased as imageFilter.

**Review by mbostock — COMMENTED** [review]
Looks good, but you can add a tiny test too?

**Review by mbostock — APPROVED** [review]

**Pull request merged**